README.md

```diff
@@ -34,7 +34,8 @@ If you use this software in an academic work, please cite:
 
 ## Installation notes
 
-DBoW2 requires [DLib](https://github.com/dorian3d/DLib), which is automatically installed if it cannot be found in the system. You can also find it in [my repository](https://github.com/dorian3d/DLib).
+ 
```

Comment:
You can also make an update here of the comments
https://github.com/dorian3d/DBoW2/blob/master/include/DBoW2/DBoW2.h#L22.
Replace requirements by ```OpenCV libraries & the boost::dynamic_bitset class.```